app/src/Services/BackupSchemaManifestService.php

```
@@ -0,0 +1,78 @@
+
+            $constraints = [];
+            foreach ($schema->constraints() as $constraintName) {
+                $constraints[$constraintName] = $this->normalizeDefinition(
+                    $schema->getConstraint($constraintName) ?? [],
+                );
+            }
+
+            $indexes = [];
+            if (method_exists($schema, 'indexes')) {
+                foreach ($schema->indexes() as $indexName) {
+                    $indexes[$indexName] = $this->normalizeDefinition($schema->getIndex($indexName) ?? []);
```

PHPStan (2.2.2)

[error] Cannot call method getIndex() on class-string|object.


(method.nonObject)